salt/modules/win_iis.py

```diff
@@ -211,8 +211,8 @@ def create_site(name, sourcepath, apppool='', hostheader='',
         name (str): The IIS site name.
         sourcepath (str): The physical path of the IIS site.
         apppool (str): The name of the IIS application pool.
-        hostheader (str): The host header of the binding. Usually the hostname
-            or website name, ie: www.contoso.com
+        hostheader (str): The host header of the binding. Usually the hostname,
+            ie: contoso.com
```

Comment:
This seemed more clear to me in its original form, tbh. This makes it less clear that you're referring to the FQDN versus just the domain name.